test/commands/contents.test.js

```diff
@@ -179,7 +179,7 @@ describe('contents command handler', () => {
 	test('when contents handler is called with valid <file> and <search> and <repo list> values, a list of repositories are logged', async () => {
 		nockScope.get(`/${repo}/contents/Procfile`).reply(200, {
 			type: 'file',
-			content: base64Encode('web: n-cluster server/init.js'),
+			content: base64Encode('web: node --max-http-header-size=80000 server/cluster.js'),
```

Comment:
**nitpick**: Great that we're getting rid of n-cluster references! perhaps we should change this to `web: node --max-http-header-size=80000 server/app.js` following the content in next-front-page/Procfile

Reply:
thanks! I picked `server/cluster.js` as it is the replacement for `n-cluster` - next-front-page doesn't use clusters. so it was like-for-like..but I'm not totally sure why the clustered example was used in the first place.